src/app/components/error-pages/not-found-page/not-found-page.component.html

```diff
@@ -0,0 +1,8 @@
+<app-mainframe>
```

Comment:
same html as access-denied-page, just a different message. You could only use 1 component that has a message input in both cases